### PR TITLE
Clarifications around timeout of participant.

### PR DESCRIPTION
**Pull request by rdebusscher**

### CODE REVIEW

**Comment by rdebusscher** [comment]
@xstefank FYI

File: spec/src/main/asciidoc/microprofile-lra-spec.adoc [hunk 1]
```diff
@@ -1010,13 +1010,16 @@ example, to indicate that an LRA should automatically cancel after 100 milliseco
 ----
 
 Furthermore, the ability to compensate or complete may be transient capabilities of a
-service so participants can also be timed out. When the time limit is reached
+service so participants can also be timed out. When the time limit is reached of the participant,
```
Comment (xstefank) [inline spec/src/main/asciidoc/microprofile-lra-spec.adoc]:
```suggestion
service so participants can also be timed out. When the time limit of the participant is reached,
```

**Comment by rdebusscher** [comment]
@mmusgrov It doesn't relate directly to that issue.

When reviewing the spec/javadoc around timeout, I found some small issue (the `to the @Compensate **and** @Complete`) and added a few clarifications.

I'll close this one and integrate it in PR for #233 now that we have agreed on that one.

**Comment by rdebusscher** [comment]
No longer valid due to changed scope of #233

**Pull request closed without merging**